### DISCUSSION TITLE
2022 年十大科技產業趨勢

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#53] [General]
1.主動式趨動方案將成為Micro/Mini LED顯示器發展趨勢
2.AMOLED技術製程再精進與螢幕下鏡頭革新
3.晶圓代工製程革新，採用FinFET及GAA技術
4.DDR5產品將逐漸量產，NAND Flash堆疊技術將超越200層
5.G擴大SA網路切片和低延遲應用比例，將進行廣泛試驗
6.低軌衛星成全球衛星營運商新戰場，3GPP納入非地面波通訊
7.從數位孿生打造元宇宙，智慧工廠將為首發場域
8.AI運算及增加感測器數量，AR/VR全面沉浸式體驗
9.自動停車（AVP）將成熱門發展功能
10.第三代半導體朝8吋晶圓及新封裝技術發展
https://technews.tw/2021/09/16/trendforce-2022-technology-industry-pulsation/

**Comment by chia-kai-chang** [comment]
有沒有同學要找一下，目前國家的科技發展重點為何?

一方面這是公開資料，二方面國家重點發展部位跟大家未來工作求職或是創業都會有關係，可以借力使力。

**Reply by EricAgumonJou** [reply]
我只有找到最新一期國家科學技術發展計畫（民國106-109年）:業經行政院於106年9月28日通過，內容涵蓋4大目標:

1.「創新再造經濟動能」:
a.推動產業創新的數位經濟發展模式
b.強化科研成果轉化機制,促進產業創新發展
c.健全區域創新系統，維繫產業聚落成長動能
d.打造創新創業之友善環境與發展機制

2.「堅實智慧生活科技與產業」:
a.發展新農業科技，提高農產安全
b.推動精準醫療科技，維護國民健康
c.精進防災科技，減少災害衝擊
d.發展綠色科技，實現低碳永續社會
e.運用智慧感測科技，維護環境品質
f.運用資通安全科技，保障國民優質生活

3.「育才競才與多元進路」:
a.培育數位經濟跨域人才
b.加強產業科技實務人才培訓機制
c.活絡多元出路，重振高階科研人才培育
d.國際頂尖人才延攬留用(推動「玉山計畫」)

4.「強化科研創新生態體系」:
a.強化科技決策支援系統
b.完備促進科技創新發展的法制環境
c.鼓勵原創科技研發，改革學術研發成果評鑑制度
d.加強產學研合作鏈結

**Reply by HSU-YU-JEN** [reply]
另外我也有找到幾項政府的科技發展重點，分別是
「智慧機械」、「亞洲‧矽谷」、「綠能科技」、「生醫產業」、「國防產業」、「新農業」及「循環經濟」等5+2產業創新計畫

**Reply by chia-kai-chang** [reply]
> 另外我也有找到幾項政府的科技發展重點，分別是 「智慧機械」、「亞洲‧矽谷」、「綠能科技」、「生醫產業」、「國防產業」、「新農業」及「循環經濟」等5+2產業創新計畫

我看到的也是這個5+2產業創新計劃
